packages/block-editor/src/components/block-alignment-control/stories/index.story.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { BlockAlignmentControl } from '../';
+
+/**
+ * The `BlockAlignmentControl` component is used to render block alignment options in the editor. The different alignment options it provides are `left`, `center`, `right`, `wide` and `full`.
+ *
+ * If you want to use the block alignment control in a toolbar, you should use the `BlockAlignmentToolbar` component instead.
+ */
+const meta = {
+	title: 'BlockEditor/BlockAlignmentControl',
+	component: BlockAlignmentControl,
+	argTypes: {
+		value: {
+			control: { type: null },
+			defaultValue: 'undefined',
```

Comment:
[`defaultValue` is deprecated](https://storybook.js.org/docs/7/api/arg-types#defaultvalue) so let's remove it. It should not affect behavior.